src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -496,12 +506,14 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
                     _selectedMaterial.isMaterialSelectionError = true
                     _selectedMaterial.materialSelectionErrorMsg = NO_COMMIT_SELECTED
                 }
-                setFilteredWorkflows(workflows)
+                setFilteredWorkflows([...workflows])
             })
             .catch((error: ServerErrors) => {
-                showError(error)
-                _selectedMaterial.isMaterialLoading = false
-                setFilteredWorkflows(workflows)
+                if (!getIsRequestAborted(error)) {
+                    showError(error)
+                    _selectedMaterial.isMaterialLoading = false
+                    setFilteredWorkflows([...workflows])
```

Comment:
[Prod issue] Have de-structured workflows since they were not getting set and were getting update on the call of polling status.